tests/framework/utils.py

```diff
@@ -190,3 +191,9 @@ def run_cmd(cmd, ignore_return_code=False, no_shell=False):
         run_cmd_async(cmd=cmd,
                       ignore_return_code=ignore_return_code,
                       no_shell=no_shell))
+
+
+def is_amd():
```

Comment:
Something similar is also used for the `cpu_features` tests. There's a local function defined in that test suite. In order to avoid duplicate code, I would suggest to move that logic in an utility class and use it both for `test_cpu_features` and for `test_snapshot_perf`. Something like: https://github.com/serban300/firecracker/commit/9dbf11b859c23fa28e54e9ff8caef41ad329153f . Of course your code for extracting the brand string is shorter and more efficient so we should leverage it instead of the old logic.

Reply:
Thanks. I'll update the existing tests.